assets/components/ctaLink/ctaLink.jsx

```diff
@@ -31,5 +44,6 @@ export default function CtaLink(props: PropTypes) {
 CtaLink.defaultProps = {
   url: null,
   onClick: null,
+  tabIndex: 0,
```

Comment:
It is a combination of things. Before the elements were not accessible via `tab` keystrokes. In order to achieve this, the elements that build up the page should be interactive web components. We had interactive components but due to design, they had the CSS property `display: none`, therefore they were not part of the page. We changed that in 5503c9d2a66976792400b6f69994509120cc0dc4. Additionally, when the links are defined via `onClick` function, the `space bar` and the `enter` keys do not achieve the desired behaviour (act as clicks). Consequently, we added that behaviour in 31aafc83be26d3b861399385bad7cf70e39b3eba.